src/inventory_ui.cpp

```diff
@@ -2053,6 +2085,7 @@ void inventory_selector::toggle_categorize_contained()
             }
         }
         own_gear_column.order_by_parent();
+        own_gear_column.order_by_category();
```

Comment:
It seems the original behavior is to put items direct below worn containers in the alternative inventory view (`;`), and now all items are put at the bottom of the UI.

Reply:
Thanks for finding that. Will look into it.